pkg/nodeconfig/nodeconfig.go

```diff
@@ -136,9 +157,14 @@ func NewNodeConfig(clientset *kubernetes.Clientset, clusterServiceCIDR, vxlanPor
 			"creating new node config")
 	}
 
+	platformType, err := getPlatformType()
+	if err != nil {
+		return nil, errors.Wrap(err, "cannot get platform type")
+	}
+
```

Comment:
A new nodeconfig object is created every time a VM is configured/deconfigured.
The platform type will never change throughout the lifecycle of the operator so we should avoid finding this more than once.
The platform type is already part of the [instanceReconciler struct](https://github.com/openshift/windows-machine-config-operator/blob/962bd45cc0e7a9735dadc2109d65b57a95d842d2/controllers/controllers.go#L53-L54) and can be passed in when calling `NewNodeConfig`

Reply:
Done